### PR TITLE
Add support for Andorra TIN

### LINKED ISSUE #119: Add support for Andorra TIN
* https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/Andorra-TIN.pdf
* https://www.impostos.ad/images/stories/docs/NRT_Passama.pdf

Examples found on the web:
* A700071W
* A700527F
* A-700555-R
* A-700747-F
* A-700814-R
* A-701315-C
* A701485T
* A-702792-H
* A-703168-T
* A-704683-Z
* A-704834-X
* A-705 321-C
* A-706010-J
* A-707871-V
* A-710.646-J
* D059888N
* D-800044-K
* D-800383-X
* F-000429-F
* F-037945-M
* F-044646-J
* F-175669-X
* F-221117-V
* F-245998-L
* L-701412-V
* L-702597-Z
* L-706185-U
* L-707969-P
* L-709222-X
* L-709418-H
* L709811-C
* L-709869-T
* L-710605-S
* L-711019-X
* L-711063-H
* L-711847-V
* L-712255-G
* L-712456-J
* L-713298-F
* O-801585-O
* U-132950-X
* U-186013-P
* U800301-M
* U-800428-R
* U-800584-Z
* U-801585-U
* U-801663-B
* U-801667-X

(below might be placeholder examples, in other words I am not sure they are actual Andorra TIN numbers)
* A-123456-X
* L-700456-A
* F-023871-V

### PR DESCRIPTION
**Pull request by unho**
Fixes #119.

### CODE REVIEW

File: tests/test_ad_nrt.doctest [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+useful as module documentation.
+
+>>> from stdnum.ad import nrt
+
+
+Tests for some corner cases.
+
+>>> nrt.validate('U132950X')
+'U132950X'
+>>> nrt.validate('U-132950-X')
```
Comment (unho) [inline tests/test_ad_nrt.doctest]: Add test for NRT separated with whitespace instead of hyphen.
